src/Components/DataBatchContext.tsx

```diff
@@ -43,15 +45,17 @@ export const DataBatchContextProvider = connect(
       setLimit(configuredLimit)
     }
 
-    const key = [
+    const baseKeys = [
       'DataBatchContextProvider',
       content.id(),
       attribute,
       id,
       tag,
       search,
       // limit is intentionally not included in the key. Otherwise the component would show a loading spinner on every "load more" click.
```

Comment:
* [ ] FYI maybe by using `dataKey` and `batchKey` we could make the code clear enough to go without the comment.